### ISSUE TITLE
Interactive elements not reachable via keyboard without VoiceOver running [iOS]

### ISSUE THREAD

**Issue by willslone** [covid-alert-ny/covid-green-app#278]
The “Next”, “Previous”, and “Close” buttons on the “How it works” screens and the “Close” graphical buttons in the top right corner of the “Select your county”, “Gender identity”, “Age range”, “Race”, and “Ethnicity” overlays are not reachable when navigating the application via keyboard without VoiceOver running (full keyboard access without VoiceOver was added in iOS 13.5).

**Comment by helloantoine** [comment]
@willslone Can you link us to some resources on how to implement this please? Is it the case of supporting TAB, SHIFT+TAB and ENTER? Any help much appreciated.

**Comment by ckiss** [comment]
fixed, avail in next build

**Comment by willslone** [comment]
The “Next”, “Previous”, and “Close” buttons on the “How it works” screens can now be reached via keyboard alone.

The “Close” graphical buttons in the top right corner of the “Select your county”, “Gender identity”, “Age range”, “Race”, and “Ethnicity” overlays are still unreachable in the latest build.

**Comment by floridemai** [comment]
This has now been addressed.
